script.js

```diff
@@ -0,0 +1,69 @@
+// Function to save the to-do list
+const saveTodos = (todos) => {
+    try {
+        localStorage.setItem("todos", JSON.stringify(todos));
+    } catch (error) {
+        console.error("Error saving todos:", error);
+    }
+};
+
+// Function to load the to-do list
+const getTodos = () => {
+    try {
+        return JSON.parse(localStorage.getItem("todos")) || [];
+    } catch (error) {
+        console.error("Error loading todos :", error);
+    }
+};
+
+// Function to render the to-do list to the page
+const renderTodos = (todos) => {
+    const todoList = document.getElementById("todo-list");
```

Comment:
Inner HTML is a massive security risk. It basically allows you to execute whatever HTML or JavaScript you want inside your application. For example, if I type valid HTML into the title, I can render whatever I want in your app. For example, I can insert an image as my title. -5

<img width="642" alt="image" src="https://github.com/rumoravenue/rumor-react-training-a1/assets/62576642/8291106f-b4cf-4e88-8a22-11e1af66a455">